inline_scan.sh

```diff
@@ -408,9 +400,15 @@ start_vuln_scan() {
 
 start_analysis() {
     # Prepare commands for container creation & copying all files to container.
-    if [[ "${d_flag-""}" ]]; then
-        CREATE_CMD+=('-d "${IMAGE_DIGEST_SHA}"')
-    fi
+    
+    for i in "${SCAN_IMAGES[@]}"; do
+        # Fetch the individual digest for each succesful image and add it to the list of digests
+        IMAGE_DIGEST=$(docker image inspect "$i" -f "{{.RepoDigests}}" | cut -f2 -d "@" | cut -f1 -d "]")
+        IMAGE_DIGEST_SHA=$IMAGE_DIGEST        
+    done
```

Comment:
If only one image can be analyze at a given time, wouldn't make sense to replace the loop to use `${SCAN_IMAGES[0]}`

Reply:
This is true, but this is part of a larger refactor that we will need to do. So leaving this alone in this PR, but will address this in a separate ticket/PR at a later date.